config/routes.rb

```diff
@@ -28,7 +27,6 @@
     collection {post :sort}
   end
   resources :forms
-  resources :vendor_categories
```

Comment:
I remember, that i failed to find a working controller for this route... but after i checked, it's there..